toolkit/tools/pkg/imagecustomizerlib/imagecustomizer_test.go

```diff
@@ -39,7 +39,7 @@ func TestCustomizeImageEmptyConfig(t *testing.T) {
 	outImageFilePath := filepath.Join(buildDir, "image.vhd")
 
 	// Create fake disk.
-	diskFilePath, _, _, err := createFakeEfiImage(buildDir)
+	diskFilePath, err := createFakeEfiImage(buildDir)
```

Comment:
Does this mean we can now revert the short test change and run test with each toolkit build?

Reply:
Hopefully. Though, I might hold off for a bit, just to make sure there aren't any other race condition bugs lurking around.